src/VscPreviewExtension/package.json

```diff
@@ -70,13 +70,16 @@
     ]
   },
   "scripts": {
-    "vscode:prepublish": "node ./node_modules/vscode/bin/compile",
-    "compile": "node ./node_modules/vscode/bin/compile -watch -p ./",
+    "vscode:prepublish": "tsc -p ./",
+    "compile": "tsc -watch -p ./",
```

Comment:
VSCode has integrated with 'tsc', so exactly, we don't depend on global command 'tsc'.

---
In reply to: [108832323](https://github.com/dotnet/docfx/pull/1507#discussion_r108832323) [](ancestors = 108832323)